apps/judicial-system/web/src/components/ContextMenu/ContextMenu.tsx

```diff
@@ -31,6 +34,10 @@ export const useContextMenu = () => {
 
   const shouldDisplayWithdrawAppealOption = useCallback(
     (caseEntry: CaseListEntry) => {
+      if (!isProsecutionUser(user) && !isDefenceUser(user)) {
```

Comment:
Consider just calling isProsecutionUser once and storing the result for reuse in line 44